config/kubernetes-sigs/sig-cluster-lifecycle/teams.yaml

```diff
@@ -175,6 +175,32 @@ teams:
     - jichenjc
     - sbueringer
     privacy: closed
+  cluster-api-provider-vsphere-admins:
+    description: Admin access to the cluster-api-provider-vsphere repo
+    maintainers:
+    - spiffxp
```

Comment:
I think we should put one of the core maintainers here.  In this case @yastij

Reply:
Updated